ecstylesheets/base.css

```diff
@@ -27,4 +27,31 @@
 .fs-07{
     font-size: 0.7rem;
 }
+.seperator{
+    border-bottom: 1px solid #494b4c;
+}
+
+.padtb-1{
```

Comment:
use better naming convention like py-1 this represents padding along y-axis as 1rem